README.md

```diff
@@ -26,58 +28,33 @@
 5. Install and update [Android Studio](https://developer.android.com/studio/index.html)
 * React Native Android installation [instructions](https://facebook.github.io/react-native/docs/getting-started.html)
 
-## Installation:
-
-1. Clone this repo
-
-2. In your terminal, change your directory to the root of the sample project you want to run.
-
-3. Run the command:
- 
-        npm install
-    to install required node modules
-
-4. For iOS:
-  * `cd ios`
-  * `pod install`
-
-4. For Android:
-  * `react-native link`
-  *  Add the following to your project's `build.gradle` file: 
+## Installation
 
-            maven {
-                url "http://tokbox.bintray.com/maven"
-            }
+To install this package in your project, run the following command:
 
-## Configuring the application
-
-Before running the application, you need to configure it to use the API key for your OpenTok project, along with an OpenTok session ID and token. For development purposes, you can obtain a session ID and token by navigating to your [TokBox account](https://tokbox.com/account/#/) page, selecting a project, and scrolling to the bottom of the page where it says `Generate Token`.
-
-Open the `src/App.js` file in your project and set the `this.apiKey`, `this.sessionId`, and `this.token` values to the API key, session ID, and token you obtained from your TokBox account:
 ```
-// Set Credentials
-this.apiKey = '';    // Add your API key.
-this.sessionId = ''; // Add the session ID.
-this.token = '';     // Add the token.
+npm install opentok-react-native --save
+react-native link opentok-react-native
```

Comment:
I think you moved these to the individual projects i.e BasicVideoChat and Signaling. We won't need them in the main readme

Reply:
Yes, that's right.